### PR TITLE
Bump swagger client version and use Makefile to generate

### PR DESCRIPTION
**Pull request by zhouzhuojie**
Make the upgrade of rbflagr idempotent!

The only change is the `Makefile`, other files are all auto-generated.

Next time we can easily do `make gen`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,37 +1,70 @@
 # rbflagr
```
Comment (rstratton) [inline README.md:1]: Should the use of `Makefile` be documented here?

**Comment by jasondoc3** [comment]
Have we thought about using `rake` instead?

File: README.md (hunk 1 above)
Comment (zhouzhuojie) [reply README.md:1]: This file will be auto generated every time; I can put the doc in Makefile and append them here

**Review by zhouzhuojie — COMMENTED** [review]
It's just some bash commands, and I want to make it consistent across all other clients

File: README.md (hunk 1 above)
Comment (rstratton) [reply README.md:1]: I didn't realize the `README` was autogenerated, I don't have strong feelings about the need to document the Makefile given that fact (the Makefile is simple enough anyway).

**Review by jasondoc3 — APPROVED** [review]

**Pull request merged**